### DISCUSSION TITLE
Add a "Close All Tabs" in context menu of the Home Tab

### DISCUSSION THREAD

**Discussion by giwul70** [sumatrapdfreader/sumatrapdf#4309] [Ideas]
**Up front**: maybe there is a faster way that I do not know of.

Right now, when multiple PDF files are open, to close all I proceed as follows:
1. select the 1st PDF (next to the Home-tab)
2, context menu, close all  tabs to the right
3. close the 1st PDF
Suggestion: add a kind of  right-click "Close all" to the Home tab, so as to close all pdf files and revert to the startup screen of SumatraPDF.

Like below.
(Again, I am not aware of a faster way, maybe there is)

Thanks!

![SumatraPDF-Suggestion-Add Close All to the home tab-08062024 070035](https://github.com/sumatrapdfreader/sumatrapdf/assets/101186455/71863d2e-50ba-4b1a-8d17-da6d174e9037)

**Comment by GitHubRulesOK** [comment]
3.6 Pre-release close all tabs
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/dd35b9f4-5d44-4b8e-aeb8-7f0801f52f31)
closes all and thus only home shown untabbed
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/3a264b06-9b6e-42f4-9b5f-0cb4b3456211)

**Reply by giwul70** [reply]
**Wow!**

Super.
Many thanks indeed.
